apps/web/app/[lang]/blog/components/SuggestedPost.tsx

```diff
@@ -4,8 +4,11 @@ import { PostList } from "./PostList";
 
 export const SuggestedPost = () => {
   return (
-    <div className="pt-12">
-      <h3 className="text-2xl font-bold mb-2">Seleccion del Editor</h3>
+    <div className="flex-col justify-start items-start gap-8 inline-flex">
+      <div
```

Comment:
Please add a TODO notation to mark this as a pending translation one we add a final i18n implementation
```suggestion
      // TODO: i18n
      <div
```